orb/src/jobs/heroku-promote.yml

```
@@ -12,3 +22,5 @@ steps:
   - run:
       name: Deploy to production
       command: npx dotcom-tool-kit deploy:production
+  - change-api/generate:
+      systemCode: <<parameters.node-version>>
```

Suggestion: I think this should be systemCode?
 ```suggestion
       systemCode: <<parameters.systemCode>>